Fix: add board overlay for esp32c3_supermini (Closes zephyrproject-rtos/zephy/#83487)

elasticdotventures opened the pull request:
Fixes #83487

elasticdotventures committed:
Fix: add board overlay for esp32c3_supermini (Closes zephyrproject-rtos/zephy/#83487)

rruuaanng commented on samples/basic/blinky_pwm/boards/esp32c3_supermini.overlay:
```diff
@@ -0,0 +1,44 @@
+};
+
+&pinctrl {
+	ledc0_default: ledc0_default {
```
```suggestion
	ledc0_default: ledc0-default {
```

rruuaanng commented on samples/basic/blinky_pwm/boards/esp32c3_supermini.overlay:
```diff
@@ -0,0 +1,44 @@
+
+	pwmleds {
+		compatible = "pwm-leds";
+		pwm_led_blue: pwm_led_gpio0_2 {
```
```suggestion
		pwm_led_blue: pwm-led-gpio0-2 {
```

rruuaanng replied:
It's just a suggestion, If you want, you don't have to follow it.

rruuaanng commented:
@elasticdotventures And, you should to change commit info. the format of the commit information is usually as follows:

```
subsys_name: ... : title

commit body info

Signed-off-by: firstname lastname <email>
```

elasticdotventures committed:
Update samples/basic/blinky_pwm/boards/esp32c3_supermini.overlay

Co-authored-by: RUANG (James Roy) <longjinyii@outlook.com>

elasticdotventures committed:
Update samples/basic/blinky_pwm/boards/esp32c3_supermini.overlay

Co-authored-by: RUANG (James Roy) <longjinyii@outlook.com>

elasticdotventures committed:
samples: blinky: add esp32 supermini overlay

Adds an overlay file to support the ESP32 supermini board in the blinky sample.

Signed-off-by: Brian Horakh <brianh@elastic.ventures>

elasticdotventures commented:
> @elasticdotventures will you be able to come back to this pull request? Cheers!

@kartben & @rruuaanng thanks!

elasticdotventures committed:
Merge branch 'main' into main

kartben commented:
was fixed with #85670
